### PR TITLE
WI-42885 Updated stubs with 7.3.0 changes (partly)

### PR DESCRIPTION
**Pull request by aalwash**
WI-42885 Updated stubs with 7.3.0 changes (partly)

### CODE REVIEW

File: json/json.php [hunk 1]
```diff
@@ -344,16 +347,25 @@ function json_last_error_msg () {}
 define('JSON_ERROR_UTF16',10);
 
 /**
- * @since 7.2
- **/
-define('JSON_INVALID_UTF8_IGNORE',1048576);
+ * @since 7.3
+ */
+define('PHP_JSON_THROW_ON_ERROR', 4194304);
```
Comment (andrey-sokolov) [inline json/json.php]: it should be `JSON_THROW_ON_ERROR`

Comment (andrey-sokolov) [inline json/json.php]: it should be JSON_THROW_ON_ERROR

Comment (aalwash) [reply json/json.php]: Thanks for CR, fixed

Comment (andrey-sokolov) [reply json/json.php]: Thank you for contribution and fast fix

File: json/json.php [hunk 2]
```diff
@@ -344,16 +347,25 @@ function json_last_error_msg () {}
 define('JSON_ERROR_UTF16',10);
 
 /**
- * @since 7.2
- **/
-define('JSON_INVALID_UTF8_IGNORE',1048576);
```
Comment (Majkl578) [inline json/json.php]: Where did this constant go? It doesn't seem to be removed in 7.3.

File: json/json.php [hunk 3]
```diff
@@ -344,16 +347,25 @@ function json_last_error_msg () {}
...
+ * json_decode() or json_encode() and causes these functions to throw a
+ * JsonException upon an error, instead of setting the global error state that
+ * is retrieved with json_last_error(). JSON_PARTIAL_OUTPUT_ON_ERROR takes
+ * precedence over <b>JSON_THROW_ON_ERROR</b>.
+ * </p>
+ *
+ * @since 7.3.0
+ * @link https://wiki.php.net/rfc/json_throw_on_error
  */
-define('JSON_INVALID_UTF8_SUBSTITUTE',2097152);
```
Comment (Majkl578) [inline json/json.php]: Same here.

Comment (aalwash) [reply json/json.php]: Both constants where already in that json file, so I removed the duplicates

**Pull request merged**